packages/gestalt/src/Upsell.js

```diff
@@ -99,11 +101,23 @@ export default function Upsell({
           mdMarginBottom={primaryLink || secondaryLink ? 0 : undefined}
           mdPaddingY={3}
         >
+          {image && (
+            <Box
+              marginBottom={0}
+              marginTop={0}
+              mdMarginBottom="auto"
```

Comment:
Nice CSS trick!

Might be good to also think about what we should do on smaller screens
![image](https://user-images.githubusercontent.com/127199/100883155-dcc1e280-3464-11eb-9ac1-75eade78db78.png)